pkgs/development/compilers/llvm/common/bintools.nix

```diff
@@ -1,49 +1,62 @@
-{ lib, runCommand, stdenv, llvm, lld, version, release_version }:
+{
+  lib,
+  runCommand,
+  stdenv,
+  llvm,
+  lld,
+  version,
+  release_version,
+}:
 
 let
-  targetPrefix = lib.optionalString (stdenv.hostPlatform != stdenv.targetPlatform) "${stdenv.targetPlatform.config}-";
+  targetPrefix = lib.optionalString (
+    stdenv.hostPlatform != stdenv.targetPlatform
+  ) "${stdenv.targetPlatform.config}-";
 in
 runCommand "llvm-binutils-${version}"
-{
-  preferLocalBuild = true;
-  passthru = {
-    isLLVM = true;
-    inherit targetPrefix;
-  };
-}
-  (''
-    mkdir -p $out/bin
-    for prog in ${lld}/bin/*; do
-      ln -s $prog $out/bin/${targetPrefix}$(basename $prog)
-    done
-    for prog in ${llvm}/bin/*; do
-      ln -sf $prog $out/bin/${targetPrefix}$(basename $prog)
-    done
+  {
+    preferLocalBuild = true;
```

Comment:
Just curious, what's the reasoning behind making this a local build preference?

Reply:
Same reasoning as in the [trivial builders](https://github.com/NixOS/nixpkgs/blob/2ab53fad627302ae55a11548f85e5a653b42136d/pkgs/build-support/trivial-builders/default.nix), I believe: building the derivation is cheap enough (we're just making symlinks) that building this derivation on a remote machine is likely to be slower (extra I/O).

Reply:
Oh, wouldn't it be better to use `runCommandLocal` instead of `runCommand` with this attribute applies?